plugin/src/test/groovy/com/novoda/staticanalysis/internal/LogsConfigurationIntegrationTest.groovy

```diff
@@ -0,0 +1,88 @@
+package com.novoda.staticanalysis.internal
+
+import com.novoda.test.Fixtures
+import com.novoda.test.TestProject
+import com.novoda.test.TestProjectRule
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+
+import static com.novoda.test.LogsSubject.assertThat
+
+@RunWith(Parameterized.class)
+class LogsConfigurationIntegrationTest {
+
+    private static
+    final String DEFAULT_CHECKSTYLE_CONFIG = "checkstyle { configFile new File('${Fixtures.Checkstyle.MODULES.path}') }"
+
+    @Parameterized.Parameters
+    public static List<Object[]> rules() {
```

Comment:
Can't you just have `List` here? Does it have to be List of Object array?
Not 100% sure but I would expect following to work.

```
@Parameterized.Parameters
public static List rules() {
    [TestProjectRule.forJavaProject(), TestProjectRule.forAndroidProject()]
}
```